finetune/target_models/masked_language_model.py

```diff
@@ -12,6 +12,43 @@
 from tensorflow.data import Dataset
 
 
+def get_mask(seq_len, config):
+    mask_proba = config.mask_proba / config.mask_spans
+
+    a = np.random.rand(seq_len)
+    if config.table_mask_bias:
+        # init_mlm_mask = a < np.flip(np.arange(0, mask_proba * 2, mask_proba * 2/seq_len))
+        init_mlm_mask = a < [.5] * 20 + [mask_proba / 2] * (seq_len - 20)
```

Comment:
Had a hard time wrapping my head around this one -- could we just wrap the right side of the comparison in a ()?

Reply:
I honestly though the `<` operator would apply before the *, +

Reply:
So If I understand this correctly, this will bias towards masking the first 20 tokens?

Reply:
yep!